### PR TITLE
Imported data layer updates

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: ts/Core/Utilities.ts [hunk 1]
```diff
@@ -207,34 +207,45 @@ function merge<T>(): T {
...
         ret = {} as T;
-    const doCopy = function (copy: any, original: any): any {
-        // An object is replacing a primitive
-        if (typeof copy !== 'object') {
-            copy = {};
-        }
 
-        objectEach(original, function (value, key): void {
+    // describtive error stack:
+    const copyDepthError = new Error('Recursive copy depth > 100'),
```
Comment (TorsteinHonsi) [inline ts/Core/Utilities.ts:214]: Shouldn't we stick to our own [error handling practice](https://github.com/highcharts/highcharts/blob/c09c2bb8ddcd8bece69a074bc8d3e3fef7e7eb61/ts/Core/Utilities.ts#L83)?

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Unless there's a specific reason, I think we should use consistent error handling - see inline comment.

File: ts/Core/Utilities.ts (hunk 1 above)
Comment (bre1470) [reply ts/Core/Utilities.ts:214]: Good point. This is from Dashboards 2020 chart component and can be removed.

**Review by TorsteinHonsi — APPROVED** [review]
Looks good, thanks!

**Pull request merged**